vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -88,11 +85,9 @@ switch ($windowsSKU) {
             "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.23.11", # for k8s 1.23.x
             "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.24.3", # for k8s 1.24.x
             # OMS-Agent (Azure monitor). Owner: ganga1980 (Ganga Mahesh Siddem)
-            "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod10042022-3c05dd1b"
-        )
-    }
-    "2022-containerd" {
-        $global:imagesToPull = @(
+            "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod10042022-3c05dd1b")
+} elseif ($windowsSku.Contains("2022-containerd")) {
```

Comment:
Can we compare the sku strictly to avoid any human mistakes?

Reply:
ok